### PR TITLE
Upgrading versions of Python and Ansible

### PR DESCRIPTION
**Pull request by hcherukuri**

### CODE REVIEW

File: tests/sanity/ignore-2.11.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+plugins/modules/redhat_csp_download.py validate-modules:undocumented-parameter
```
Comment (sabre1041) [inline tests/sanity/ignore-2.11.txt:1]: Can we update the module to avoid as many ignored tests as possible?

**Comment by sabre1041** [comment]
@hcherukuri A couple general comments (applicable to all of the upgrades that we will be performing)

* Can we use the FQCN for all tasks? So `ansible.builtin.include_tasks` instead of `include_tasks`
* Can we generate collection documentation

File: tests/sanity/ignore-2.11.txt (hunk 1 above)
Comment (hcherukuri) [reply tests/sanity/ignore-2.11.txt:1]: I was ignoring some issues with Python 3.9 https://pastebin.com/7YBhVQ9v

**Comment by hcherukuri** [comment]
> @hcherukuri A couple general comments (applicable to all of the upgrades that we will be performing)
> 
> * Can we use the FQCN for all tasks? So `ansible.builtin.include_tasks` instead of `include_tasks`
> * Can we generate collection documentation

We usually have role documentation, in this case we don't have any roles. Do you want me to add collection installation steps from galaxy ?

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**